editors/MacVim/files/patch-python3.diff

```diff
@@ -1,11 +1,11 @@
 --- src/configure.ac	2021-04-05 15:48:50.000000000 +0200
 +++ src/configure.ac	2021-04-05 15:48:56.000000000 +0200
-@@ -1663,7 +1663,7 @@
+@@ -1688,7 +1680,7 @@ __:
  	@echo "python3_LIBS='$(LIBS)'"
  	@echo "python3_SYSLIBS='$(SYSLIBS)'"
  	@echo "python3_DLLLIBRARY='$(DLLLIBRARY)'"
 -	@echo "python3_INSTSONAME='$(INSTSONAME)'"
 +	@echo "python3_INSTSONAME='$(PYTHONFRAMEWORKPREFIX)/$(INSTSONAME)'"
+	@echo "python3_PYTHONFRAMEWORKPREFIX='$(PYTHONFRAMEWORKPREFIX)'"
```

Comment:
Actually I think this patch should no longer be needed at all. It looks like this was finally accepted/implemented upstream: https://github.com/vim/vim/commit/b6ebe5af74a965d703eb8ee66a07c97b0bbe6e58

Reply:
Thanks for the review. It does indeed appear that the `patch-python3.diff` isn't needed for MacVim any longer. I have removed the patch file and the corresponding `patchfiles-append` directive from the MacVim Portfile in thomasnyman@5bca4e078d27901f63dfaba150e134d1f99b71be.

The Vim `patch-python3.diff` is still needed for the Vim Portfile, as that now applies is selectively if building a version earlier than v9.0.1441 (which I suppose in practice only means v9.0.0472 on MacOS 10.10 and earlier due to [#67209](https://trac.macports.org/ticket/67209)).

The Vim `patch-python.diff`and `patch-tcl.diff` actually apply cleanly to v9.0.1677 which is why they are not modified in these changes.